bin/nrnpyenv.sh

```diff
@@ -132,20 +133,40 @@ if test "$z" = "Darwin" ; then
     unset p
     unset d
     unset l
-  else
+    nrnpylib_provenance="sysconfig LIBDIR"
+  fi
+  if test "$z" = "" ; then
+    z=$($p -c '
+try:
+  from neuron import h
+  s=h.libpython_path()
+  s = s if ".dylib" in s else ""
```

Comment:
`nitpick` / `minor suggestion` : in the new code, we have been recommending more descriptive variable names than single letter (for new developers it useful)

Reply:
I will do that for s,z,p,d,l

Reply:
using more meaningful names